k6 tests fails with go1.13

go1.13 is released, running k6 tests under it, the test is failing:

```
--- FAIL: TestVUIntegrationClientCerts (1.72s)
    --- FAIL: TestVUIntegrationClientCerts/Unauthenticated (0.66s)
        --- FAIL: TestVUIntegrationClientCerts/Unauthenticated/Source (0.13s)
        Error Trace:    runner_test.go:1168
        Error:      	Error message not equal:
                    	expected: "GoError: Get https://127.0.0.1:36593: remote error: tls: bad certificate"
                    	actual  : "GoError: Get https://127.0.0.1:36593: readLoopPeekFailLocked: remote error: tls: bad certificate"
        Test:       	TestVUIntegrationClientCerts/Unauthenticated/Source
        --- FAIL: TestVUIntegrationClientCerts/Unauthenticated/Archive (0.13s)
        Error Trace:    runner_test.go:1168
        Error:      	Error message not equal:
                    	expected: "GoError: Get https://127.0.0.1:36593: remote error: tls: bad certificate"
                    	actual  : "GoError: Get https://127.0.0.1:36593: readLoopPeekFailLocked: remote error: tls: bad certificate"
        Test:       	TestVUIntegrationClientCerts/Unauthenticated/Archive
time="2019-09-04T01:20:32+07:00" level=warning msg="Request Failed" error="Get http://httpbin.local:33807/html: lookup httpbin.local: no such host"
```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -2,7 +2,7 @@ version: 2
 jobs:
   deps:
     docker:
-      - image: circleci/golang:1.12
+      - image: circleci/golang:1.13
     environment:
       GOPATH: /home/circleci/.go_workspace
     working_directory: /home/circleci/.go_workspace/src/github.com/loadimpact/k6
@@ -22,7 +22,7 @@ jobs:
 
   test:
     docker:
-      - image: circleci/golang:1.12
+      - image: circleci/golang:1.13
     environment:
       GOPATH: /home/circleci/.go_workspace
     working_directory: /home/circleci/.go_workspace/src/github.com/loadimpact/k6
@@ -45,7 +45,7 @@ jobs:
 
   test-prev-golang:
     docker:
-      - image: circleci/golang:1.11
+      - image: circleci/golang:1.12
     environment:
       GOPATH: /home/circleci/.go_workspace
     working_directory: /home/circleci/.go_workspace/src/github.com/loadimpact/k6
@@ -61,7 +61,7 @@ jobs:
 
   build-docker-images:
     docker:
-      - image: circleci/golang:1.12
+      - image: circleci/golang:1.13
     environment:
       GOPATH: /home/circleci/.go_workspace
     working_directory: /home/circleci/.go_workspace/src/github.com/loadimpact/k6
@@ -105,7 +105,7 @@ jobs:
 
   build-linux-packages:
     docker:
-      - image: circleci/golang:1.12
+      - image: circleci/golang:1.13
     environment:
       GOPATH: /home/circleci/.go_workspace
     working_directory: /home/circleci/.go_workspace/src/github.com/loadimpact/k6

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -10,7 +10,7 @@ environment:
   # specific to go
   VERSION: "%APPVEYOR_REPO_TAG_NAME:v=%"
   GOPATH: c:\gopath
-  GOVERSION: 1.12.1
+  GOVERSION: 1.13
   GOMAXPROCS: 2
   CGO_ENABLED: '0'
   GOARCH: amd64

--- a/js/runner_test.go
+++ b/js/runner_test.go
@@ -1165,7 +1165,8 @@ func TestVUIntegrationClientCerts(t *testing.T) {
 				vu, err := r.NewVU(make(chan stats.SampleContainer, 100))
 				if assert.NoError(t, err) {
 					err := vu.RunOnce(context.Background())
-					assert.EqualError(t, err, fmt.Sprintf("GoError: Get https://%s: remote error: tls: bad certificate", listener.Addr().String()))
+					require.NotNil(t, err)
+					assert.Contains(t, err.Error(), "remote error: tls: bad certificate")
 				}
 			})
 		}

